1-Initial-Setup/1.8-GNOME-Display-Manager.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+@test "1.8.1 Ensure GNOME Display Manager is removed (Manual)" {
+    run bash -c "dpkg -s gdm3 | grep -E '(Status:|not installed)'"
+    [ "$status" -eq 1 ]
```

Comment:
```suggestion
    [ "$status" -eq 1 ]
    [[ "$output" == *"dpkg-query: package 'gdm3' is not installed and no information is available"* ]]
```

Reply:
I think checking the exit code is already sufficient. The output check could go wrong on machines which are not configured to use the English language.

Reply:
Bump